### PR TITLE
give instructions for how to ping the ICE-breaker teams

### PR DESCRIPTION
**Pull request by nikomatsakis**

### CODE REVIEW

**Comment by nikomatsakis** [comment]
r? @Mark-Simulacrum, this is correct, right?

cc @spastorino

File: src/ice-breaker/about.md [hunk 1]
```diff
@@ -47,3 +47,17 @@ cargo run add-person $your_user_name
...
 
 * [Example of adding yourself to the LLVM ICE-breakers.](https://github.com/rust-lang/team/pull/140)
+
+## Tagging an issue for an ICE-breaker group
+
+To tag an issue as appropriate for an ICE-breaker group, you give
+[rustbot] a [`ping`] command with the name of the ICE-breakers
+team. For example:
+
+```
```
Comment (JohnTitor) [inline src/ice-breaker/about.md]:
```suggestion
```text
```

make ci happy

File: src/ice-breaker/about.md [hunk 2]
```diff
@@ -47,3 +47,17 @@ cargo run add-person $your_user_name
 Example PRs:
 
 * [Example of adding yourself to the LLVM ICE-breakers.](https://github.com/rust-lang/team/pull/140)
+
+## Tagging an issue for an ICE-breaker group
+
+To tag an issue as appropriate for an ICE-breaker group, you give
+[rustbot] a [`ping`] command with the name of the ICE-breakers
```
Comment (lzutao) [inline src/ice-breaker/about.md]: Do [rustbot] and [`ping`] labels miss links?

Comment (nikomatsakis) [reply src/ice-breaker/about.md]: oops yes

**Pull request merged**